### DISCUSSION TITLE
Statistics and more like in Zoraxy

### DISCUSSION THREAD

**Discussion by ConGp20** [NginxProxyManager/nginx-proxy-manager#4138] [Ideas]
It would be great, if there would be some features that zoraxy has, like statistics, country blocks and more. I like npm, it works great for a long time in my setup. i am very happy that npm is crowdsec-supported, unlike zoraxy

**Comment by jaycedk** [comment]
Try this https://github.com/xavier-hernandez/goaccess-for-nginxproxymanager
